docs/manpage.md

```diff
@@ -129,6 +129,7 @@ Some parts of booster boot functionality can be modified with kernel boot parame
    If debug level is enabled then kmsg throttling gets disabled automatically.
  * `booster.debug` an obsolete option that is equivalent to `booster.log=debug,console`.
  * `quiet` Set booster init verbosity to minimum. This option is ignored if `booster.debug` or `booster.log` is set.
+ * `modules` a comma-separated list of extra kernel modules which should be force loaded.
```

Comment:
what parameter name is used by other initramfs? I vaguely remember dracut likes to use `rd.` prefix for the parameters used by initramfs only.

Reply:
* Alpine's mkinitfs uses `modules`, which is why I ended up using it here as well but I also wouldn't mind using something else of cause
* dracut seems to distinguish `rd.driver.pre` and `rd.driver.post`: https://github.com/dracutdevs/dracut/blob/master/man/dracut.cmdline.7.asc#misc
* mkinitcpio seems to use `earlymodules`: https://wiki.archlinux.org/title/Mkinitcpio#init_from_base_hook

Reply:
I propose to use the dracut's idea with `rd.` prefix for initrd-specific parameters. I am going to merge your change to `wip` and modify the parameter name.

Reply:
Or maybe even call it `rd. modules_force_load` to match the name to config option that does the same.